lib/mac_layer/pdu/mpdu.cpp

```diff
@@ -92,7 +94,9 @@ namespace ex2
       }
       catch (MPDUHeaderException& e) {
         // @todo should log this
+//#if MPDU_DEBUG
         printf("MPDUHeader exception : %s\n", e.what());
+//#endif
         throw MPDUException("MPDU: Bad raw MPDUHeader.");
```

Comment:
I think this exception should be caught somewhere too. Or is it? I want to avoid unhandled exceptions in this library, TI doesn't like them very much